ci/check-protobuf-stability.sh

```diff
@@ -89,26 +89,33 @@ USAGE
   exit
   ;;
 --stable)
-  if [[ $TARGET == "main" ]]; then
-    # Check against the most recent stable tag.
-    # This check runs only on main or PRs targeting main.
-    #
-    # This check does not need to run on release branch commits because
-    # they are built sequentially, so no conflicts are possible and the per-PR
-    # check is enough.
-    readonly LATEST_STABLE_TAG="$(git tag | grep -v "snapshot" | sort -V | tail -1)"
-    # The v1.17 stable release includes the buf config file with the default name `buf.yml`.
-    # Starting with v1.18 we have multiple buf config files.
-    if [[ $LATEST_STABLE_TAG =~ "v1.17."* ]]; then
-      BUF_CONFIG_UPDATED=false
-    else
-      BUF_CONFIG_UPDATED=true
-    fi
-    BUF_GIT_TARGET_TO_CHECK=".git#tag=${LATEST_STABLE_TAG}"
-    check_protos
+  # Check against the most recent stable tag.
```

Comment:
```suggestion
  # Check against the highest stable tag.
```

Pretty sure we had a period last year where 1.11.3 was the most recent one, even though 1.17.1 existed.